core-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -105,15 +109,20 @@ public void statusChanged(boolean login) {
           HttpRequestInitializer initializer = new HttpRequestInitializer() {
             @Override
             public void initialize(HttpRequest httpRequest) throws IOException {
+              HttpHeaders headers = new HttpHeaders();
+              headers.setUserAgent(USER_AGENT);
```

Comment:
Cloud repo syncing happens, I think, in com.google.gct.idea.git.GcpCheckoutProvider (correct me if I'm wrong about that.) This uses completely different classes so we'll need different code to set the user agent though we can reuse the calculation of the user agent string.